airbyte-config/models/src/main/resources/types/ReplicationAttemptSummary.yaml

```diff
@@ -9,17 +9,40 @@ required:
   - bytesSynced
   - startTime
   - endTime
+  - totalRecordsEmitted
```

Comment:
why are we not creating a stat object that can be an aggregation or per stream?
```
totalStats: {
   recordEmitted,
   bytesEmitted,
   stateMessagesEmitted,
}
streamStats: {
  abc: {
     recordEmitted,
     bytesEmitted,
     stateMessagesEmitted,
   },
   def: {
      recordEmitted,
      bytesEmitted,
      stateMessagesEmitted, # that will work with the v2 as we split the state per stream
   }
}
```

Reply:
Great call, made this change and ran a local test sync, summary is much cleaner:

```
io.airbyte.config.ReplicationAttemptSummary@518962dd[
  status=completed,
  recordsSynced=450,
  bytesSynced=6651,
  startTime=1641580813211,
  endTime=1641580869369,
  totalStats=io.airbyte.config.SyncStats@5eb70e1[
    recordsEmitted=450,
    bytesEmitted=6651,
    stateMessagesEmitted=50,
    recordsCommitted=450
  ],
  streamStats=io.airbyte.config.StreamNameToSyncStats@2af96d68[
    additionalProperties={
      data3=io.airbyte.config.SyncStats@3de87200[
        recordsEmitted=150,
        bytesEmitted=2217,
        stateMessagesEmitted=<null>,
        recordsCommitted=150
      ], 
      data2=io.airbyte.config.SyncStats@276d60ea[
        recordsEmitted=150,
        bytesEmitted=2217,
        stateMessagesEmitted=<null>,
        recordsCommitted=150
      ],
      data1=io.airbyte.config.SyncStats@79df709e[
        recordsEmitted=150,
        bytesEmitted=2217,
        stateMessagesEmitted=<null>,
        recordsCommitted=150
      ]
    }
  ]
]

```